Remove outdated getComputedStyle tests for text-decoration

nt1m opened the pull request:
#### ad22399c1297d4acbfc8f79dee05fde251f68bae
<pre>
Remove outdated getComputedStyle tests for text-decoration
<a href="https://bugs.webkit.org/show_bug.cgi?id=296112">https://bugs.webkit.org/show_bug.cgi?id=296112</a>
<a href="https://rdar.apple.com/156027078">rdar://156027078</a>

Reviewed by Vitor Roriz.

...and move the backwards compatibility ones as WPT subtests.

* LayoutTests/fast/css/getComputedStyle/getComputedStyle-text-decoration-expected.txt: Removed.
* LayoutTests/fast/css/getComputedStyle/getComputedStyle-text-decoration.html: Removed.
* LayoutTests/fast/css3-text/css3-text-decoration/getComputedStyle/getComputedStyle-text-decoration-shorthand-expected.txt: Removed.
* LayoutTests/fast/css3-text/css3-text-decoration/getComputedStyle/getComputedStyle-text-decoration-shorthand.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-valid.html:

Canonical link: <a href="https://commits.webkit.org/297538@main">https://commits.webkit.org/297538@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/8022618257e509d424c2ca6d97cbdd8bdaa82c97

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/112044 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/31715 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/22200 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/118067 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/62284 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/32391 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/40292 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/85115 "Passed tests") | [❌ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/35778 "") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/114991 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/25870 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/100817 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/65548 "Passed tests") | 
| | [⏳ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/iOS-18-Simulator-WPT-WK2-Tests-EWS "Waiting to run tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/18957 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/61919 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/95253 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/19032 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/121386 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/39077 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/29075 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/93953 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/39458 "Built successfully") | [![loading](https://user-images.githubusercontent.com/3098702/171232313-daa606f1-8fd6-4b0f-a20b-2cb93c43d19b.png) 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/97070 "Build is in progress. Recent messages:Running configuration; Skipping applying patch since patch_id isn't provided; Running checkout-pull-request; Running run-layout-tests-in-stress-mode") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/93771 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/23948 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/38983 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/16774 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/35099 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/38971 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/38608 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/41936 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/40324 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/b0eb51deb7d0e4aad95a9d9a8784c33b8f328653. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48171

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8c7fcbc0d87fd3acbbadd194c113fb335b8490c9. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48171

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/8022618257e509d424c2ca6d97cbdd8bdaa82c97. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/48171

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-computed.html:
```diff
@@ -30,6 +30,10 @@
 test_computed_value("text-decoration", "underline dashed rgb(0, 255, 0)");
 
+// Test backwards compatibility of blink.
+test_computed_value("text-decoration", "underline overline line-through blink", ["underline overline line-through", "underline overline line-through blink"]);
```
Discussed this offline and serialization is not clear for these values atm, so I'd rather not add more to this Interop test until the CSSWG has discussed this.

nt1m committed:
Remove outdated getComputedStyle tests for text-decoration
https://bugs.webkit.org/show_bug.cgi?id=296112
rdar://156027078

Reviewed by Vitor Roriz.

...and move the backwards compatibility ones as WPT subtests.

* LayoutTests/fast/css/getComputedStyle/getComputedStyle-text-decoration-expected.txt: Removed.
* LayoutTests/fast/css/getComputedStyle/getComputedStyle-text-decoration.html: Removed.
* LayoutTests/fast/css3-text/css3-text-decoration/getComputedStyle/getComputedStyle-text-decoration-shorthand-expected.txt: Removed.
* LayoutTests/fast/css3-text/css3-text-decoration/getComputedStyle/getComputedStyle-text-decoration-shorthand.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-computed-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-computed.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-valid-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/css/css-text-decor/parsing/text-decoration-valid.html:

Canonical link: https://commits.webkit.org/297538@main

webkit-commit-queue commented:
Committed 297538@main (ad22399c1297): <https://commits.webkit.org/297538@main>

Reviewed commits have been landed. Closing PR #48171 and removing active labels.
